network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1386,20 +1397,30 @@ def get_sitemap_urls(self, request):
 
     def get_context(self, request, *args, **kwargs):
         context = super().get_context(request, *args, **kwargs)
+        language_code = self.get_language_code(request)
 
         authenticated = request.user.is_authenticated
-        key = 'home_product_dicts_authed' if authenticated else 'home_product_dicts_live'
+        key = (
+            f"home_product_dicts_authed_{language_code}"
+            if authenticated
+            else f"home_product_dicts_live_{language_code}"
+        )
+
```

Comment:
feels like we can keep the original, then follow up with `key = f'{key}_{language_code}'`